packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -260,14 +293,15 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget>
     final prevRootId = id(_properties?.node);
     final newRootId = id(getRoot(selectedNode));
     final shouldFetch = prevRootId != newRootId;
-    FlexLayoutProperties newSelection = _properties;
     if (shouldFetch) {
-      newSelection = await fetchFlexLayoutProperties();
+      final newSelection = await fetchFlexLayoutProperties();
+      _setProperties(newSelection);
+    } else {
+      _setProperties(FlexLayoutProperties.fromDiagnostics(selectedNode));
```

Comment:
this should be _setProperties(_properties) because `FlexLayoutProperties.fromDiagnostics(selectedNode)` will return null that was caused by selectedNode not containing the render object.